### ISSUE TITLE
Server crash on Linux Build 1244

### ISSUE THREAD

**Issue by Dhannyjsb** [KRILLE123/esx-kr-advanced-shops#18]
Ca't start server on linux 1244 with esx-kr-advanced-shops, 

> esx-kstart esx-kr-advanced-shops
Creating script environments for esx-kr-advanced-shops
Stacktrace:

/proc/self/maps:
40a68000-40b38000 rwxp 00000000 00:00 0
41549000-41559000 rwxp 00000000 00:00 0
41fde000-42113000 rwxp 00000000 00:00 0
2a1ac580000-2a1ac600000 rw-p 00000000 00:00 0
51aad980000-51aada00000 rw-p 00000000 00:00 0
71002100000-71002180000 rw-p 00000000 00:00 0
78d22200000-78d22280000 rw-p 00000000 00:00 0
797c9900000-797c9a0a000 rw-p 00000000 00:00 0
88cd0480000-88cd0500000 rw-p 00000000 00:00 0
caacb980000-caacba00000 rw-p 00000000 00:00 0
d6f25880000-d6f25900000 rw-p 00000000 00:00 0
ffeb7000000-ffeb7080000 rw-p 00000000 00:00 0
cfx> 108cbe480000-108cbe488000 rw-p 00000000 00:00 0
115f08400000-115f08480000 rw-p 00000000 00:00 0
12ac56b00000-12ac56b80000 rw-p 00000000 00:00 0
15b2a0a80000-15b2a0b00000 rw-p 00000000 00:00 0
18812b400000-18812b480000 rw-p 00000000 00:00 0
1963be700000-1963be780000 rw-p 00000000 00:00 0
1c5e3f880000-1c5e3f900000 rw-p 00000000 00:00 0
2100a0e80000-2100a0f00000 rw-p 00000000 00:00 0
21cf31043000-21cf31080000 ---p 00000000 00:00 0
21cf31080000-21cf31083000 rw-p 00000000 00:00 0
21cf31083000-21cf31084000 ---p 00000000 00:00 0
21cf31084000-21cf310ff000 r-xp 00000000 00:00 0
21cf310ff000-21cf31100000 ---p 00000000 00:00 0

Unhandled Exception:
System.NullReferenceException: Object reference not set to an instance of an object
GlobalError: Unhandled exception in Mono script environment: System.NullReferenceException: Object reference not set to an instance of an object
(null)
Unhandled exception in Mono script environment: System.NullReferenceException: Object reference not set to an instance of an object

**Comment by chasticfy** [comment]
yeah same here with a build of 1265 en 1244 it crashes yes 1265 no it's not out yet

**Comment by lostreboy** [comment]
Try commentout or disable fonts in the resource.lua ;)
